### PR TITLE
Updates to Schemas and Flask for ActivitySchedule Documents

### PR DESCRIPTION
**Pull request by anantmittal**

### CODE REVIEW

**Comment by anantmittal** [comment]
@jayfo Should we add `valueId` as required field in `activity-schedule.json`? To avoid situations where we have activity-schedule.json referencing orphan activity.json with null/undefined `valueId`.

**Comment by anantmittal** [comment]
`test_patients_get` test in test_registry_patients.py gets an assertion failure because of mismatch between new values inventory jsonschema and existing values inventory document (matches old values inventory schema) in database.

**Review by jayfo — APPROVED** [review]

**Comment by jayfo** [comment]
Confirmed all tests pass with a reset database.

File: scope_shared/scope/testing/fake_data/fixtures_fake_activity.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ def _fake_activity(
                 datetime_now,
             )
         ),
+        "valueId": "valueId Placeholder",
```
Comment (anantmittal) [inline scope_shared/scope/testing/fake_data/fixtures_fake_activity.py:35]: @jayfo valueId is an optional field.

Comment (jayfo) [reply scope_shared/scope/testing/fake_data/fixtures_fake_activity.py:35]: Fixed in #361.

**Pull request merged**